proto/backend.proto

```diff
@@ -321,9 +321,6 @@ service AdmissionControl {
 
   // Return a modified copy of the request that can be saved or a descriptive error
   rpc MutateAdmission(AdmissionRequest) returns (MutationResponse);
-
-  // Convert a resource to a new version
-  rpc ConvertObject(ConversionRequest) returns (ConversionResponse);
```

Comment:
@andresmgot @wbrowne how should i handle these lint failures?

Reply:
These checks aren't required to merge and are primarily there (IIRC) to ensure people are aware/conscious of when breaking changes are being made. We use the [FILE category for breaking detection](https://buf.build/docs/breaking/rules) so we know when clients will break. Since these services aren't used yet (even though plugins will have been built with this capability and Grafana is setup to call them), it _should_ be safe. It wouldn't hurt to do some manual validation though.

I'd say if you want to be extra careful then just add the new stuff and just deprecate the other instead if possible:
```protobuf
  // Convert a resource to a new version
  rpc ConvertObject(ConversionRequest) returns (ConversionResponse) {
    option deprecated = true;
  };
```

Reply:
good call. i'll test this out with grafana and a plugin using the previous SDK version before merging. i'll add the deprecated method if there are any issues